src/sonic-yang-models/tests/yang_model_tests/tests_config/ntp.json

```diff
@@ -4,13 +4,38 @@
             "sonic-ntp:NTP_SERVER": {
                 "NTP_SERVER_LIST": [
                     {
-                        "server_address": "10.11.12.13"
+                        "server_address": "10.11.12.13",
+                        "association_type": "server",
+                        "iburst": "on",
+                        "key": 10,
+                        "admin_state": "enabled",
+                        "trusted": "no"
                     },
                     {
-                        "server_address": "2001:aa:aa::aa"
+                        "server_address": "2001:aa:aa::aa",
+                        "association_type": "server",
+                        "iburst": "off",
+                        "key": 15,
+                        "admin_state": "disabled",
+                        "trusted": "yes"
                     },
                     {
-                        "server_address": "pool.ntp.org"
+                        "server_address": "pool.ntp.org",
+                        "association_type": "pool",
+                        "iburst": "on",
+                        "admin_state": "enabled"
+                    }
+                ]
+            },
+            "sonic-ntp:NTP_KEY": {
+                "NTP_KEYS_LIST": [
+                    {
+                        "id": 10,
+                        "value": "lumos"
```

Comment:
SONiC Yang shows value is encrypted, does the test only accept plain text value?

Reply:
it is for tests only. I will change it to encrypted value to avoid confusions